modular_skyrat/modules/colony_fabricator/code/machines/solid_fuel_generator.dm

```diff
@@ -0,0 +1,59 @@
+/obj/machinery/power/port_gen/pacman/solid_fuel
+	name = "solid fuel generator"
```

Comment:
Feels kind of boring as a name, compared to `P.A.C.M.A.N.`. Might be worth trying to find a slightly more enticing name.

Reply:
What do you think of S.O.F.I.E. (Stationary Operating Fuel Ignition Engine)

Reply:
I can get behind that, yeah.